_data/nav.yml

```diff
@@ -666,10 +667,9 @@
   - url: "php-upload-handler"
   - url: "editor-command-identifiers"
     pages:
-      - url: "#Listing core and plugin editor commands"
-      - url: "#Core Editor commands"
-      - url: "#Plugin Commands"
-      - url: "#Editor Management Commands"
+      - url: "#Executable commands"
+      - url: "#Query command states"
+      - url: "#Query value commands"
```

Comment:
`Query command values`? As this and state really should be consistent.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1590/commits/966fedd38b68c8897d01d21eefeb077e14ea954c